angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/nimble-table-column-anchor.directive.ts

```diff
@@ -0,0 +1,139 @@
+    }
+
+    @Input() public set rel(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'rel', value);
+    }
+
+    public get target(): string | undefined {
+        return this.elementRef.nativeElement.target;
+    }
+
+    @Input() public set target(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'target', value);
+    }
+
+    public get type(): string | undefined {
+        return this.elementRef.nativeElement.type;
+    }
+
+    @Input() public set type(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'type', value);
+    }
+
+    public get download(): string | undefined {
+        return this.elementRef.nativeElement.download;
+    }
+
+    @Input() public set download(value: string | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'download', value);
+    }
+
+    public get fractionalWidth(): number | null | undefined {
+        return this.elementRef.nativeElement.fractionalWidth;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('fractional-width') public set fractionalWidth(value: NumberValueOrAttribute | null | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'fractionalWidth', toNullableNumberProperty(value));
+    }
+
+    public get minPixelWidth(): number | null | undefined {
+        return this.elementRef.nativeElement.minPixelWidth;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('min-pixel-width') public set minPixelWidth(value: NumberValueOrAttribute | null | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'minPixelWidth', toNullableNumberProperty(value));
+    }
+
+    public get groupIndex(): number | null | undefined {
+        return this.elementRef.nativeElement.groupIndex;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('group-index') public set groupIndex(value: NumberValueOrAttribute | null | undefined) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'groupIndex', toNullableNumberProperty(value));
+    }
+
+    public get groupingDisabled(): boolean {
+        return this.elementRef.nativeElement.groupingDisabled;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('grouping-disabled') public set groupingDisabled(value: BooleanValueOrAttribute) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'groupingDisabled', toBooleanProperty(value));
+    }
```

Comment:
We should define mixins for these apis similar to how we have done for the table column authors so that the directive properties can be consistent.
It could be a follow-up PR if preferred

Reply:
I started trying to adapt the mixin pattern used with the table column types, but I got stuck. I'll try to address this in a follow-up PR, but I'll probably need some help with it.

Reply:
Can you create a tech debt item to track it (good practice to not accidentally forget the follow-up PRs we want)

Reply:
Created issue #1211